app/build.gradle

```
@@ -91,6 +91,9 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.4'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.0'
+
+    //dot indicator
```

```suggestion
    // dot indicator
```

별 거 아니지만 ktlint 문법대로 주석을 써봅시다 🙃